client/my-sites/subscribers/components/subscriber-list-actions-bar/subscriber-list-actions-bar.tsx

```diff
@@ -26,10 +27,14 @@ const ListActionsBar = () => {
 		setSortTerm,
 		filterOption,
 		setFilterOption,
+		siteId,
 	} = useSubscribersPage();
 	const sortOptions = useMemo( () => getSortOptions( translate ), [ translate ] );
 	const recordSort = useRecordSort();
-	const filterOptions = useSubscribersFilterOptions();
+	const { data: subscribersTotals = { total: 0 } } = useSubscribersTotalsQueries( siteId );
+	const totalSubscribers = subscribersTotals?.total ?? 0;
+	const skipAll = totalSubscribers > 30000; // 30000 is the limit of subscribers that can be fetched without breaking the endpoint. This is a temporal solution.
```

Comment:
I feel like we need to move this 3000 to a constant. Mainly since it's used in multiple places.

Reply:
True 👍